### PR TITLE
View Products Front End | Elliot Park

### PR DESCRIPTION
**Pull request by elliooo**
**Changes:**
- Added ProductList and ProductListItem components
- Changed App render method to render a ProductList

**Screenshot:**
Screen Shot 2020-01-13 at 1 17 24 PM (2)

### CODE REVIEW

File: client/components/app.jsx [hunk 1]
```diff
@@ -1,25 +1,18 @@
...
-  componentDidMount() {
-    fetch('/api/health-check')
-      .then(res => res.json())
-      .then(data => this.setState({ message: data.message || data.error }))
-      .catch(err => this.setState({ message: err.message }))
-      .finally(() => this.setState({ isLoading: false }));
-  }
+  // constructor(props) {
+  //   super(props);
+  // }
```
Comment (thebearingedge) [inline client/components/app.jsx]: Please do not turn in commented out code.

File: server/public/styles.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
```
Comment (thebearingedge) [inline server/public/styles.css]: Is this CSS all necessary? `box-sizing` is already set by Bootstrap. What exactly do you want `margin: 0` on? Typically if you want to influence all of the text, you'd just set the `body` font. Are you sure you need to?

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: client/components/app.jsx (hunk 1 above)
Comment (elliooo) [reply client/components/app.jsx]: Resolved in c2d14c1.

File: server/public/styles.css (hunk 2 above)
Comment (elliooo) [reply server/public/styles.css]: Resolved in c2d14c1.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Your header could use some bottom margin

**Comment by elliooo** [comment]
**Updated Screenshot:**
Screen Shot 2020-01-13 at 2 56 19 PM (2)

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**